### PR TITLE
Removes "header()" to send as HTML

### PR DESCRIPTION
**Pull request by lucanos**
This was causing a PHP error as the output body had already commenced at Line 1

### CODE REVIEW

**Comment by seblucas** [comment]
Can you share some informations, I cannot reproduce it on my end.

Thanks in advance

**Comment by seblucas** [comment]
@lucanos 

ping ;)

**Comment by lucanos** [comment]
Rerun your test with error reporting switched on ("E_ALL").
Otherwise, the error is probably being produced, but muted, by your setup.

**Comment by seblucas** [comment]
Currently I'm running PHP7 with : 

```
error_reporting = E_ALL | E_STRICT
```

and I have no errors (in the log or on the screen)

**Comment by seblucas** [comment]
Same with PHP 5.4.14 and PHP 5.6.24

**Comment by seblucas** [comment]
Do you have a closing `?>` in your `config_local.php` ?

**Comment by lucanos** [comment]
Look at the file I made the change to.

Lines 1 & 2 are output to the browser - it's the HTML5 Doctype declaration, and the start of the HTML element.

You can't use the PHP header() command after you have started sending body content to the client. That is why it gave me an error. I have no idea why it isn't doing so for you, but it should be.

> Remember that header() must be called before any actual output is sent, either by normal HTML tags, blank lines in a file, or from PHP. It is a very common error to read code with include, or require, functions, or another file access function, and have spaces or empty lines that are output before header() is called. The same problem exists when using a single PHP/HTML file.

http://php.net/manual/en/function.header.php

**Comment by seblucas** [comment]
Let me rephrase, I know about this limitation and I know you're right and this PR will eventually be merged ;).

I'm just trying to understand why I don't have this problem to make sure it does not happen elsewhere / again

**Comment by seblucas** [comment]
Found it, I had this line in my php.ini : 

```
output_buffering = 4096
```

Changing it to `Off` show the error you have.

**Comment by seblucas** [comment]
Thanks a lot.
